### ISSUE TITLE
The results of the network have black blocks in the lower right corner of the image

### ISSUE THREAD

**Issue by Ivan-Zhao** [jmiller656/EDSR-Tensorflow#14]
The results of the network have black blocks in the lower right corner of the image

**Comment by PrathamSoni** [comment]
This is due to an error in the model.py under the predict method. When the dimensions of the original image do not line up with the "img_size" the bottom right corner is not calculated. add this code to the predict method to fix the issue 
`if (x.shape[0]%self.img_size != 0 and  x.shape[1]%self.img_size != 0):`
				`tmp = self.sess.run(self.out,feed_dict={self.input:[x[-1*self.img_size:,-1*self.img_size:]]})[0]`
				`tmp_image[-1*tmp.shape[0]:,-1*tmp.shape[1]:] = tmp`

**Comment by Ivan-Zhao** [comment]
Another problem，how to set learning rate？

**Comment by jmiller656** [comment]
@PrathamSoni Thanks again, please view my comment on your pull request.

@Ivan-Zhao learning rate can be set in the code where AdamOptimizer is initialized. If you'd like, I can create a command line argument for it instead

**Comment by matt-dong** [comment]
I am also having this issue but I fixed it with the above code. This is an example of the issue.
![outoutput](https://user-images.githubusercontent.com/32016991/34083529-239866f8-e340-11e7-9333-0df10b664fcd.png)

**Comment by PrathamSoni** [comment]
Fix was merged.
